.env.example

```
@@ -0,0 +1,5 @@
+WORDPRESS_DB_HOST=host.docker.internal:3307
+WORDPRESS_DB_USER=dbuser
+WORDPRESS_DB_PASSWORD=dbpassword
+WORDPRESS_DB_NAME=dbname
+WORDPRESS_TABLE_PREFIX=wp_
```

```suggestion
WORDPRESS_TABLE_PREFIX=wp_

```
1. Missing new line
2. Why do we need to specify table prefix?